server/integration/feature-pack/src/main/resources/content/bin/jolokia.bat

```diff
@@ -1,5 +1,5 @@
 set AGENT_BOND_JAR=
-for /r %JBOSS_HOME%\modules\io\fabric8\agent-bond\main %%X in (*.jar) do (
+for /r %JBOSS_HOME%modules\system\add-ons\@infinispan.module.slot.prefix@\io\fabric8\agent-bond\@infinispan.module.slot@ %%X in (*.jar) do (
```

Comment:
Hrmm, can we assume that `JBOSS_HOME` env variable will always have the separator present?

Reply:
Nope, that was removed by mistake

Reply:
Fixed